Peer.Domain/Models/ConfigModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Peer.Domain.Models
+{
+    public class ConfigModel
+    {
+        public GithubConfig Github { get; set; } = new();
```

Comment:
Why initialize them to empty configs? Seems like something you might want to make either optional (in case the user doesn't have one of the accounts), something that is injected separately (GitHubConfig, AzDoConfig) and registered separately. Or something that is correct by construction and has a ctor that ensures that things are in the appropriate states

Reply:
Fixed.